manifests/integreatly-solution-explorer/solution-explorer-0.0.48/solution-explorer-operator.v0.0.48.clusterserviceversion.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+  name: solution-explorer-operator.v0.0.48
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - kind: WebApp
+        name: webapps.integreatly.org
+        version: v1alpha1
+        displayName: SolutionExplorer
+        description: SolutionExplorer
+  description: Placeholder description
+  displayName: Tutorial Web App Operator
+  install:
+    spec:
+      deployments:
+        - name: tutorial-web-app-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: tutorial-web-app-operator
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  name: tutorial-web-app-operator
+              spec:
+                containers:
+                  - command:
+                      - tutorial-web-app-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: OPERATOR_NAME
+                        value: tutorial-web-app-operator
+                    image: quay.io/dimitra/tutorial-web-app-operator:v0.0.48
```

Comment:
Leaving my org in here until https://github.com/integr8ly/tutorial-web-app-operator/pull/89 is merged